src/aat/resources/features/F-121/S-121.6.td.json

```diff
@@ -772,14 +774,15 @@
 									"label": null,
 									"hint_text": null,
 									"show_condition": null,
-									"retain_hidden_value": null
-								},
+									"retain_hidden_value": null,
+								"retain_hidden_value": null},
 								{
 									"complex_field_element_id": "RespondentDetails.RespAddress3.County",
 									"display_context": "HIDDEN",
 									"label": null,
 									"hint_text": null,
 									"show_condition": null,
+                  "retain_hidden_value": null,
```

Comment:
how come there's 2 `retain_hidden_value`?

Reply:
Looks like this happened during a merge. Resolved.